packages/@aws-cdk/cx-api/lib/yaml.ts

```diff
@@ -0,0 +1,95 @@
+import * as yaml from 'yaml';
```

Comment:
I'm not sure I support putting this stuff in `cx-api`. Doesn't seem to belong there.

I guess a new library?

Reply:
Done. I called it `@aws-cdk/yaml-cfn`.